playbooks/templates/k8s/jail/matomo-jail.yml.j2

```
@@ -35,14 +35,16 @@ spec:
       - name: copy
         image: {{ wai_matomo_image }}:{{ 'latest' if item == 'wai-stag' else wai_matomo_image_tag }}
         imagePullPolicy: {{ 'Always' if item == 'wai-stag' else 'IfNotPresent' }}
-        command: ["sh", "-c", ". /entrypoint.sh"]
+        command: ["sh", "-c", ". /entrypoint.sh", "{{ matomo_name }}"]
```

Non dovrebbe essere fatta in questo modo `command: ["sh", "-c", ". /entrypoint.sh {{ matomo_name }}"]`

---

Penso che il risultato sia lo stesso, perché in ogni caso vengono ignorati i valori `ENTRYPOINT` e `CMD` dell'immagine.
Stando alla [documentazione di K8S](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#run-a-command-in-a-shell), la versione più "corretta" dovrebbe essere:
```suggestion
        command: ["sh"]
        args: ["-c", ". /entrypoint.sh {{ matomo_name }}"]
```